File: Python/Korekta.py
d = int(input())
for _ in range(d):
    stas_win, stas_loss, wies_win, wies_loss, grz_win, grz_loss = map(int, input().split())
    total_games = stas_win + stas_loss
    print((stas_loss + wies_loss + grz_loss) - total_games, (stas_win + wies_win + grz_win) - total_games)

File: Java/Korekta.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int d = input.nextInt();
        while (d-- > 0) 
        {
            int stasWin = input.nextInt();
            int stasLoss = input.nextInt();
            int wiesWin = input.nextInt();
            int wiesLoss = input.nextInt();
            int grzWin = input.nextInt();
            int grzLoss = input.nextInt();
            int totalGames = stasWin + stasLoss;
            System.out.println(((stasLoss + wiesLoss + grzLoss) - totalGames) + " " + ((stasWin + wiesWin + grzWin) - totalGames));
        }
        input.close();
    }
}